### DISCUSSION TITLE
Partial text search not working in table blocks?

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#5999] [Help]
I am trying out Table block that has built-in searchable columns:

![Screenshot 2022-05-20 at 4 41 57 PM](https://user-images.githubusercontent.com/1195573/169490075-16c666e2-03cd-49a2-a308-9107aaedaac0.jpg)

If I type full text "Client 1 Headquarter Block B" in the search field, the filter works correctly:

![Screenshot 2022-05-20 at 4 43 33 PM](https://user-images.githubusercontent.com/1195573/169490577-65768eed-d778-4835-bf0f-e0be74861557.jpg)

However, if I type partial text, like "Headquarter", it returns no results:

![Screenshot 2022-05-20 at 4 43 41 PM](https://user-images.githubusercontent.com/1195573/169490420-f62f37a1-30fb-4fc2-acb7-fa8cb2f36918.jpg)

Is this a bug? How to achieve partial search?

**Comment by magickaito** [comment]
After reading other discussions, I learnt that - the built in search fields are using "start with" logic, the search term must match the starting characters. Even if I build my own text fields to try the "LIKE" operator, partial text search will also not work as "contains search is not yet supported" as mentioned here https://github.com/Budibase/budibase/discussions/1874

**Reply by melohagan** [reply]
Hi @magickaito you're right. The 'like' operator does not work for the internal tables: https://github.com/Budibase/budibase/issues/3386

**Reply by magickaito** [reply]
ok got it! T___T
